### PR TITLE
Rewriting alerts best practices doc

### PR DESCRIPTION
**Pull request by kaileyhaynes**
Rewrote the "Alerts best practices" doc to be more applicable to those practices.

Jira ticket here: [#7101](https://newrelic.atlassian.net/browse/DOC-7101)

### CODE REVIEW

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 1]
```diff
@@ -16,36 +16,86 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
 ---
 
-This document describes some best practices and examples to help you get the most out of your alerts configuration.
+Improve your alert coverage by implementing the following recommendations.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:19]: alert --> alerts

Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:19]: I feel like you threw the baby out with the bath water here :)

I'd love to see a combo of what was here before and the new thing you've got. (Hint: I like the first part of your sentence and the second part of the old.)

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 2]
```diff
@@ -16,36 +16,86 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
 ---
 
-This document describes some best practices and examples to help you get the most out of your alerts configuration.
+Improve your alert coverage by implementing the following recommendations. 
+
+Read on to learn the best practices for:
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:21]: In general, I'd love to see the section headers in this doc get more descriptive. For example, instead of "Policies" something like "Get the most out of your policies" or even just "Organize your policies". Get some verbs in there to make it easier to skim those section headers.

Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:21]: Once you've rewritten the section headers, I think line 21 could be rewritten. Something like.... er.... "Use these links to jump to the topic you're interested in learning more about" or something... 

It's also possible that you could lose this whole links section (it's in the right nav) once you've rewritten those section headers. Something to think about :)

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 3]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+- [Policies](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#policy-practices)
+- [Notification channels](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#channel-practices)
+- [Incident preferences](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#incident-practices)
+- [Thresholds and violations](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#threshold-practices)
+- [Muting roles](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#mute-practices)
 
 <Callout variant="important">
-  Before reading this, we recommend you read [Alerts concepts and workflow](/docs/alerts/new-relic-alerts/getting-started/alert-policy-workflow).
+  Before reading this, we recommend you also read [Alerts concepts and workflow](/docs/alerts/new-relic-alerts/getting-started/alert-policy-workflow).
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:29]: I'm super cold on this callout. It's kind of a drag to immediately send someone somewhere else :) Consider deleting!

If you want to keep it, and I think there are arguments for that, then I think it would be helpful to add some information about why we think someone should read that other doc.

Something like: "This doc describes how the different parts of alerts work together to notify you about problems in your system." or something

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 4]
```diff
@@ -16,36 +16,86 @@ redirects:
...
 </Callout>
 
-## Define policies for entities or people [#entities-people]
+## Recommended alerts [#recommend-alerts]
+
+Use [recommended practices](https://discuss.newrelic.com/t/announcing-alert-recommendations-for-apm-applications/154542) if you are new to Alerts or if you want suggestions that optimize your alert coverage.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:34]: I got called out for this last week :) In general when we're linking to something outside of the docs site, we want to call out. 

We have standard wording when linking to the Explorers Hub. "see our [Explorers Hub post on recommended alerts practices]()"

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 5]
```diff
@@ -16,36 +16,86 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
 ---
 
-This document describes some best practices and examples to help you get the most out of your alerts configuration.
+Improve your alert coverage by implementing the following recommendations. 
+
+Read on to learn the best practices for:
+- [Policies](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#policy-practices)
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:22]: Also, the order of sections in this doc feels a bit weird to me.

I'll need to think a bit more on this...

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 6]
```diff
@@ -16,36 +16,86 @@ redirects:
...
 </Callout>
 
-## Define policies for entities or people [#entities-people]
+## Recommended alerts [#recommend-alerts]
+
+Use [recommended practices](https://discuss.newrelic.com/t/announcing-alert-recommendations-for-apm-applications/154542) if you are new to Alerts or if you want suggestions that optimize your alert coverage.
+
+## Policies [#policy-practices]
+
+A policy is a container for alike conditions.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:38]: I like how simple this is. I keep getting thrown by the word "alike" though. Maybe it's just me? :) I prefer "similar".

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 7]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+
+<Callout variant="tip">
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:40]: I'm torn by these callouts. I think they're potentially pretty useful, but it also leads to a lot of callouts :)

I do like putting "If you're new to alerts" at the start. What would these look like without a callout?

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 8]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+Learn how to [create, edit, or find policies](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) if you’re new to Alerts.
+</Callout>
 
-When designing your policies, consider:
+Organize your policy by surrounding its conditions to a single entity and to the specific team that needs to be notified when a problem arises within that entity.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:44]: Watch your prepositions. In "conditions to a single entity", for example, "to" feels odd there. Especially related to the word "surrounding". Definitely check that in the rest of the doc too.

I do really like the impulse behind this rewrite, though.

Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:44]: Think about how someone might respond to this sentence if they're unfamiliar with any of these terms: "policy", "condition", "entity".

Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:44]: In general, throughout the alerts docs, there's a lot of consideration for the technical detail, but... a lot less consideration for the reader. Our ongoing task is to think about how we can swap that emphasis and focus: consider the reader, but also provide enough technical detail for these docs to still be meaningful.

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx (hunk 3 above)
Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:29]: Also, if we're sending someone somewhere else as prerequisite knowledge, what would it look like to fold that content into this doc? 

To be clear, I'm not asking you to literally do that :) More of a thought experiment.

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 9]
```diff
@@ -16,36 +16,86 @@ redirects:
...
 
-* The parts of your architecture that need personnel to be responsible for them
-* The individuals who are responsible for one or more parts of your infrastructure
+If a team is monitoring several groups of the same entity type, combine those entity clusters (like servers) together into one policy. This way, your team can be notified from one policy rather than navigating several policies at once.
 
-An organization may have multiple entities monitored by APM, browser, infrastructure, and synthetic monitoring. Examples of considerations for different teams:
+Consider your team’s role when assigning them to policies:
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:48]: I think you may be losing some useful detail in this rewrite....

Also, it's not the "team's" role that we want to consider, right? More the individual on a team? Or maybe it's a combination of team and individual roles?

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 10]
```diff
@@ -16,36 +16,86 @@ redirects:
...
 
 * **Software developers** may need [notifications](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-controlling-where-send-alerts) for both front-end and back-end performance, such as webpage response time and page load JavaScript errors.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:50]: Would be curious to know if there are other roles we're not considering... but that's a much larger question. This reminds me of an internal post I saw a while back....

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 11]
```diff
@@ -16,36 +16,86 @@ redirects:
...
 * **Operations personnel** may need notifications for poor back-end performance, such as server memory and load averages.
 * **The product owner** may need notifications for positive front-end performance, such as improved end user Apdex scores or sales being monitored in [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-new-relic-one-dashboards).
 
-By following these best practices, key personnel will receive actionable notifications for the metrics that matter to them, and overall, the organization will be able to identify and respond to trends or patterns more efficiently.
+Refrain from creating policies that include multiple teams to keep policies centralized.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:54]: There's something about "refrain" that bothers me a bit. There's a bit of a hectoring tone to it.

I prefer to present a more positive face on stuff...

How about something more like:

"Although you can create policies that include multiple teams, we recommend policies focused on specific teams. That way you won't be spamming unrelated people unnecessarily." I don't totally love this either, but I think you get what I'm going for?

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 12]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+
+## Notification channels [#channel-practices]
+
+Tailor notifications to the most useful channel and policy so you can avoid alert fatigue and help the right personnel receive and respond to incidents they care about in a systematic way.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:58]: Tailor is good

Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:58]: I like this sentence a lot

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 13]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+
+<Callout variant="tip">
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:60]: I almost put this in a parenthetical instead of a callout... :D I don't really think we should do that though

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 14]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+Learn how to [set up notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/) if you’re new to Alerts.
+</Callout>
+
+Notify teams and individuals who:
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:64]: This might be my own personal aesthetic, but I always try to avoid bullets when I only have two of 'em.

You've got "when an incident arises" twice, which makes me think you could collapse this into a single sentence pretty easily.

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 15]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+- Can resolve a problem when an incident arises
+- Need to stay updated when an incident arises
+
+For staying updated, select a notification channel that is less intrusive like email.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:68]: I prefer "to stay updated"

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 16]
```diff
@@ -16,36 +16,86 @@ redirects:
...
 
-## Control how many notifications you get [#more-or-less]
+For vital notifications and responding to the incident, select a notification channel that is immediate and reliable like PagerDuty or Slack. You are given a lot more control over who gets notified and when with PagerDuty and OpsGenie.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:70]: I prefer "incident responses" over "responding to the incident"

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 17]
```diff
@@ -16,36 +16,86 @@ redirects:
...
-## Control how many notifications you get [#more-or-less]
+For vital notifications and responding to the incident, select a notification channel that is immediate and reliable like PagerDuty or Slack. You are given a lot more control over who gets notified and when with PagerDuty and OpsGenie. 
 
-The more [conditions](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions) you define, the more [incidents](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/explore-incident-history) can be triggered and monitored. For example, your organization may need an alerting solution to accommodate extensive IT systems. Create policies with multiple conditions for [multiple monitored entities](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/select-product-targets-alert-condition) that notify you through one or more notification channels. Set [incident preference](/docs/alerts/new-relic-alerts/configuring-alert-policies/specify-when-new-relic-creates-incidents) to determine how violations lead to notifications.
+Do not rely on email for quick notifications in case of mail delays. An email list will also bounce if one is no longer valid.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:72]: "mail delays" --> "delays"

Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:72]: Also, it's not so much the email list that will bounce, but individual email addresses on the email list, yeah?

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx (hunk 16 above)
Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:70]: I'm a little unclear on the purpose of line 70. 

Are you saying that there are two types of notification channels?
Immediate and reliable (PagerDuty/Slack) versus more control over who gets notified (PagerDuty/OpsGenie)? Or ...?

Instead of emphasizing these other products/companies, I'd turn that focus back on alerts.

Something more like: "Alerts integrates with a wide variety of notification channels, depending on your needs. These channels include: PagerDuty, Slack, etc etc" (don't use the etcs :D ) 

I don't know much about these other products, but someone who is already using it probably just wants to know that we support it.

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 18]
```diff
@@ -16,36 +16,86 @@ redirects:
...
 
-On the other hand, your organization may not need an extensive alerting structure. The fewer conditions you define and the more minimal your incident preference is, the fewer violations will be opened. For example, for a simple alerting solution, you could create a simple policy with only an email [notification channel](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-controlling-where-send-alerts).
 
-## Set thresholds for conditions [#conditions]
+## Incident preferences [#incident-practices]
 
-Set the [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-thresholds-trigger-alert) for your policy's conditions to meaningful levels for your environment. Here are some suggested guidelines:
+Decide when you get incident notifications so you can respond to incidents when they happen.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:77]: Yes! This is pretty much perfect.

I sort of want to change "when they happen" to "when you want to". Or something like that. Like, let's emphasize that the power is in their hands :)

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 19]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+
+<Callout variant="tip">
+Learn more about your [incident preferences options](https://discuss.newrelic.com/t/relic-solution-alert-incident-preferences-are-the-key-to-consistent-alert-notifications/40867) if you’re new to Alerts.
+</Callout>
+
+The default incident preference setting combines all conditions within a policy into one incident. Change your default incident preference setting to better understand the exact scope and number of all violations.
+
+Each organization will have different needs, and each policy within an organization will have different needs too. Ask your team two important questions when deciding your incident preferences:
+- Do we want to be notified every time something goes wrong?
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:86]: Is there are a third bullet you can think of? I just have this hang up for not having just two bullets :D

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 20]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+
+<Callout variant="tip">
+Learn more about your [incident preferences options](https://discuss.newrelic.com/t/relic-solution-alert-incident-preferences-are-the-key-to-consistent-alert-notifications/40867) if you’re new to Alerts.
+</Callout>
+
+The default incident preference setting combines all conditions within a policy into one incident. Change your default incident preference setting to better understand the exact scope and number of all violations.
+
+Each organization will have different needs, and each policy within an organization will have different needs too. Ask your team two important questions when deciding your incident preferences:
+- Do we want to be notified every time something goes wrong? 
+- Do we want to group all alike notifications together and be notified once?
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:87]: There's that "alike" again. I'm not sure what my problem is with it, but it might just be me haha

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 21]
```diff
@@ -16,36 +16,86 @@ redirects:
...
-## Set thresholds for conditions [#conditions]
+## Incident preferences [#incident-practices]
 
-Set the [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-thresholds-trigger-alert) for your policy's conditions to meaningful levels for your environment. Here are some suggested guidelines:
+Decide when you get incident notifications so you can respond to incidents when they happen.
+
+<Callout variant="tip">
+Learn more about your [incident preferences options](https://discuss.newrelic.com/t/relic-solution-alert-incident-preferences-are-the-key-to-consistent-alert-notifications/40867) if you’re new to Alerts.
+</Callout>
+
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:82]: "Change your default incident preference setting to better understand the exact scope and number of all violations." I'm not clear on how changing my default incident preference will help me better understand. 

I feel like there's something in between the two halves of this sentence. A missing logical piece.

1. Change default preferences
2. ???
3. Better understand 

See? :)

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 22]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+The default incident preference setting combines all conditions within a policy into one incident. Change your default incident preference setting to better understand the exact scope and number of all violations.
+
+Each organization will have different needs, and each policy within an organization will have different needs too. Ask your team two important questions when deciding your incident preferences:
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:85]: I'd change the focus a little:

"Each team within your organization may have different alerting needs..." or something like that

"Each organization" feels too distant and impersonal

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 23]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+- Do we want to be notified every time something goes wrong? 
+- Do we want to group all alike notifications together and be notified once?
+
+When a policy and its conditions have a broader scope (like managing several entities), increase your incident preference sensitivity. You will need more notifications because two incidents will not necessarily relate to each other.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:89]: It's not so much "managing several entities" as "monitoring the performance of several entities", yeah?

Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:89]: I think you need a little more on why you should increase your incident preference sensitivity. I mean, I think it's because you don't want to get too many notifications, right? 

The more things you're alerting on, the more notifications you can potentially get.

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 24]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+
+When a policy and its conditions have a focused scope (like managing one entity), opt for one notification. You will need less notifications when two incidents are related to each other or when the team is already notified and fixing an existing problem.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:91]: It should be, opt for one policy, yeah?

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 25]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+
+Decide how you get incident notifications by using our best [notification channel practices](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#channel-practices).
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:93]: I'd rephrase that to "notification channel best practices"

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 26]
```diff
@@ -16,36 +16,86 @@ redirects:
...
+
+
+## Thresholds and violations [#threshold-practices]
+
+Set meaningful [threshold](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-thresholds-trigger-alert) levels to optimize Alerts to your business. Here are some suggested guidelines:
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:98]: "optimize alerts for your business"

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 27]
```diff
@@ -106,21 +156,22 @@ Set the [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-polici
   </tbody>
 </table>
 
-In most of our products (except Infrastructure), the color-coded [health status indicator](https://docs.newrelic.com/docs/accounts-partnerships/education/getting-started-new-relic/glossary#health-status) in the user interface changes as the alerting threshold escalates or returns to normal. This allows you to monitor a situation through our UI before a critical threshold passes, without needing to receive specific notifications about it.
+In most of our products (except Infrastructure), the color-coded [health status indicator](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#health-status) in the user interface changes as the alerting threshold escalates or returns to normal. This allows you to monitor a situation through our UI before a critical threshold passes, without needing to receive specific notifications about it.
+ 
+There are two violation thresholds: critical (red) and warning (yellow). Define these thresholds with different criteria keeping in mind the suggestions above.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:161]: "...different criteria, keeping in mind..."

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 28]
```diff
@@ -106,21 +156,22 @@ Set the [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-polici
...
+In most of our products (except Infrastructure), the color-coded [health status indicator](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#health-status) in the user interface changes as the alerting threshold escalates or returns to normal. This allows you to monitor a situation through our UI before a critical threshold passes, without needing to receive specific notifications about it.
+ 
+There are two violation thresholds: critical (red) and warning (yellow). Define these thresholds with different criteria keeping in mind the suggestions above. 
 
-For example, you can define a critical (red) threshold that notifies you when the error percentage for your app is above 10 percent at least once in any five minute period. You can also define an optional warning (yellow) threshold with different criteria.
-
-## Select notification channels [#channels]
+<Callout variant='important'>
+Warning violations do **not** open incidents. A critical violation can open incidents, but you must define that decision through your [incident preferences](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#incident-practices).
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:164]: I don't think we need the bolded not.

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 29]
```diff
@@ -106,21 +156,22 @@ Set the [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-polici
...
+</Callout>
 
-You can create [notification channels](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/notification-channels-controlling-where-send-alerts) first and then assign policies to them. You can also create policies first and then assign notification channels to them. This flexibility allows you to tailor who gets notified, using the method that is most useful to them.
+## Muting roles [#mute-practices]
 
-For example, you could:
+Mute alerts during routined events such as maintenance or planned downtime.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:169]: routined --> routine

Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:169]: "...events, such as..."

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 30]
```diff
@@ -106,21 +156,22 @@ Set the [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-polici
...
 
-* Identify your operations team's Slack channel as a general level of alerting, and use the on-call PagerDuty contact as an after-hours or escalated level of alerting.
-* Create webhooks with customized messages for a variety of situations or personnel.
+You can also silence another aspect of Alerts (like a policy, a specific entity, a condition) when needed. Incidents can still be opened, but you will not be notified.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:171]: I think you should rewrite this first sentence to remove mention of alerts and not use a parenthetical.

Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:171]: I prefer "won't" to "will not"

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx (hunk 15 above)
Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:68]: I think intrusive is fine here.

I'd definitely add a comma: "less intrusive, like email."

**Review by paperclypse — CHANGES_REQUESTED** [review]
Feel free to reach out if any of my comments aren't clear or you disagree with them :) Thanks!

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx (hunk 3 above)
Comment (kaileyhaynes) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:29]: I agree--I didn't know how to keep or remove it. I'm fine deleting it.

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx (hunk 14 above)
Comment (kaileyhaynes) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:64]: That's fair. My aesthetic is coordinated balance, but I understand your point here

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx (hunk 18 above)
Comment (kaileyhaynes) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:77]: The only reason I hesitate changing to "when you want to" because my immediate thought is: we're saying that incidents are not a big enough deal that you should respond to them when they happen. It's more laissez-faire in my opinion.

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx (hunk 19 above)
Comment (kaileyhaynes) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:86]: Stay hanging

Comment (kaileyhaynes) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:86]: I say that as a powerful response and joke: I'm going to keep it as is because it breaks the text and brings attention to the important questions for the skimmer. I'm hesitant to add a non-existent third question for an aesthetic.

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx (hunk 5 above)
Comment (paperclypse) [reply src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:22]: We talked about this over Slack

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 31]
```diff
@@ -16,36 +16,42 @@ redirects:
...
+- Muting roles
 
-## Define policies for entities or people [#entities-people]
 
-When designing your policies, consider:
+## Recommended alerts [#recommend-alerts]
 
-* The parts of your architecture that need personnel to be responsible for them
-* The individuals who are responsible for one or more parts of your infrastructure
+Use recommended practices if you are new to Alerts or if you want suggestions that optimize your alert coverage. See our [Explorers Hub post on recommended alerts practices](https://discuss.newrelic.com/t/announcing-alert-recommendations-for-apm-applications/154542).
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:31]: I feel like it should read "recommended alerts conditions" instead of "recommended practices". There's also an recommended conditions doc here https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/condition-recommendations/

**Review by paperclypse — DISMISSED** [review]
I have one minor comment, but otherwise this is great! Good work!

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 32]
```diff
@@ -16,36 +16,42 @@ redirects:
...
+- Muting roles
 
-## Define policies for entities or people [#entities-people]
 
-When designing your policies, consider:
+## Recommended alerts [#recommend-alerts]
 
-* The parts of your architecture that need personnel to be responsible for them
-* The individuals who are responsible for one or more parts of your infrastructure
+Use [recommended alerts](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/condition-recommendations/) conditions if you are new to Alerts or if you want suggestions that optimize your alert coverage.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:31]: super minor, but I'd make the link text "recommended alerts conditions". Am I being too nitpicky? :)

**Review by paperclypse — APPROVED** [review]
Approved!
